### PR TITLE
Add example with nginx configuration.

### PR DESCRIPTION
**Pull request by andriisoldatenko**
I think it's better to have working version with nginx also.

### CODE REVIEW

File: nginx/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,18 @@
+upstream web {
```
Comment (mattrobenolt) [inline nginx/nginx.conf]: You don't need to define this upstream at all.

**Comment by mattrobenolt** [comment]
The only push back I'd have here is that more typically than not, you probably also want TLS, and this pushes people moreso to running just plain port 80. I was hoping to leave this bit as an excercise for the reader since there are lots of different ways to do it. Maybe you just use an ELB or something like that. If you deploy on systems like Kubernetes, you can usually just use a different "router" load balancer in which case this nginx here isn't too valuable.

**Comment by andriisoldatenko** [comment]
@mattrobenolt Yes my main goal to add SSL also.

**Comment by mattrobenolt** [comment]
Sure. And I'd just argue that's a bit out of the scope of what we should dictate here. There are lots of TLS things to configure, how to get certificates, etc. And again, would like to support cases where they already have a load balancer or some other TLS termination service which is pretty common here.

**Comment by andriisoldatenko** [comment]
@mattrobenolt I'll close PR

**Pull request closed without merging**